pygin/modules/pygin/far.py

```diff
@@ -445,28 +474,52 @@ class FileTime(datetime.datetime):
 	def __new__(self, Value = 0):
 		return datetime.datetime.utcfromtimestamp((max(Value, _EPOCH_AS_FILETIME) - _EPOCH_AS_FILETIME) / _HUNDREDS_OF_NANOSECONDS)
 
+	def value(self):
+		return int(self.replace(tzinfo=datetime.timezone.utc).timestamp() * _HUNDREDS_OF_NANOSECONDS + _EPOCH_AS_FILETIME)
+
 @unique
 class PluginPanelItemFlags(IntFlag):
 	Default                                         = 0
 	Selected                                        = bit(30)
 	ProcessDescription                              = bit(31)
 
+@unique
+class FileAttributeFlags(IntFlag):
```

Comment:
Is it needed? Attributes are not Far-, but system-specific and are already defined in os.stat.

Reply:
Removed in 5ef1f612ff8787268b8d48658e3d1e8cad9b7272.